app/models/answer.rb

```diff
@@ -11,11 +11,13 @@ class Answer < ApplicationRecord
   has_many :reactions, as: :reactable, dependent: :restrict_with_error
   has_many :comments, as: :commentable, dependent: :restrict_with_error
   has_many :credit_transactions, as: :transactable
+  has_many :abuse_reports, as: :abuseable
 
   after_commit :notify_question_author, on: :create
 
   scope :order_by_vote, -> {order(reaction_count: :desc)}
-
+  scope :published, -> { where(published: true) }
```

Comment:
@maskedXchaotic instead of playing with published bit, lets add another boolean in all content. name it 'marked_abused' default false and use that.

So when marking a content as abused, this bit should be set to true and  make it unpublished. 
content which is marked abused can not be mark published, by controller or by console